crates/pathfinder/src/bin/pathfinder.rs

```diff
@@ -40,6 +40,12 @@ async fn main() -> anyhow::Result<()> {
 Hint: Make sure the provided ethereum.url and ethereum.password are good.",
     )?;
 
+    // Pending is only supported on mainnet.
```

Comment:
```suggestion
    // Pending is only supported on mainnet not to load feeder_gateway unnecessarily.
```

Or some other reason? I will ask about this in a few weeks otherwise :)

Reply:
I though testnet should not support pending. Will double-check with StarkWare.

Reply:
Confirmed as only being used on mainnet - since pending is a work-around for the long block times on mainnet. Will add a comment to this effect.